app/models/manageiq/providers/embedded_ansible/provider/default_ansible_objects.rb

```diff
@@ -0,0 +1,64 @@
+module ManageIQ::Providers::EmbeddedAnsible::Provider::DefaultAnsibleObjects
+  extend ActiveSupport::Concern
+
+  class DefaultAnsibleObjectError < RuntimeError; end
```

Comment:
I think this should this live with the rest of our exceptions, not in this class.